.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "editor.formatOnSave": true
+}
```

Comment:
🤔 I'm generally not a fan of putting editor-specific config in repos.

Reply:
As a solution, we can add `extensions.json` file with Prettier plugin what's automatically lint. Or just delete this `settings.json`, and add pre-commit hook?

Reply:
Removed for good future

Reply:
Yeah, seems like everything in the `@electron` org is going towards Prettier, anyway.